src/features/Header/Header.jsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/button-has-type */
+import React, { useEffect, useState } from 'react'
+import Modal from 'react-modal'
+import styled from './Header.module.scss'
+import wallpaper from '../../../public/img/header-background.png'
+// eslint-disable-next-line import/extensions
+import AddMovie from '../../base/AddMovie/index.jsx'
+// eslint-disable-next-line import/extensions
+import Search from '../../base/Search/index.jsx'
+// eslint-disable-next-line import/extensions
+import SearchButton from '../../base/SearchButton/index.jsx'
```

Comment:
if you use index as I wrote before you do not need to specify what file do you need
just point on the folder with index file
```suggestion
import SearchButton from '../../base/SearchButton'
```

Reply:
Done